### DISCUSSION TITLE
Do I commit the deploy.yml?

### DISCUSSION THREAD

**Discussion by AdamKyle** [deployphp/deployer#2563] [Help needed]
I cannot find an answer any where. Since it contains sensitive data like who to login as and the server name (or ip) I assume no. But if we add all our scripts and stuff and then transfer computers ....

So do I commit this? Can I use env variables in the .yml?

Thanks :D

**Comment by antonmedv** [comment]
Yes, commit deploy.yaml. 

All sensitive stuff put in environment vars or in ssh config.
